phpunit/class-block-library-comment-template-test.php

```diff
@@ -45,8 +45,7 @@ public function setUp() {
 		);
 	}
 
-	function test_build_comment_query_vars_from_block_with_context_no_pagination() {
-		update_option( 'page_comments', false );
+	function test_build_comment_query_vars_from_block_with_context() {
```

Comment:
I changed the order of `test_build_comment_query_vars_from_block_with_context` and `test_build_comment_query_vars_from_block_with_context_no_pagination`.

As the content of both functions is pretty similar, `git` thinks I just modified the name.